core/src/test/scala/org/apache/spark/ThreadingSuite.scala

```diff
@@ -130,7 +131,7 @@ class ThreadingSuite extends SparkFunSuite with LocalSparkContext {
               Thread.sleep(100)
             }
             if (running.get() != 4) {
-              println("Waited 1 second without seeing runningThreads = 4 (it was " +
+              logInfo("Waited 1 second without seeing runningThreads = 4 (it was " +
```

Comment:
Confirmed the test is having trouble with `logInfo` here and works fine with `println`. Should I just revert this?